index.html

```diff
@@ -2435,13 +2435,20 @@ <h3>Data Schemas</h3>
         <dl>
           <dt><var>credentialSchema</var></dt>
           <dd>
-The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
+            <p>
+The value of the <code>credentialSchema</code> <a>property</a> SHOULD be one or
```

Comment:
curious why the normative downgrade. Is there a use case that can't happen if the value of `credentiSchema` property MUST be one or more data schemas?

Reply:
this was a suggestion from @TallTed's comment here https://github.com/w3c/vc-data-model/issues/950#issuecomment-1608008708


> I also note that the current VCDM spec says that `credentialSchema` "_can_ be included by an [issuer](https://www.w3.org/TR/vc-data-model-2.0/#dfn-issuers) in the [verifiable credentials](https://www.w3.org/TR/vc-data-model-2.0/#dfn-verifiable-credential) that it issues". This optionality should be reflected in the definition that follows, i.e., —
> 
> _If present_, the value of the _optional_ `credentialSchema` [property](https://www.w3.org/TR/vc-data-model-2.0/#dfn-property) MUST be one or more data schemas that provide [verifiers](https://www.w3.org/TR/vc-data-model-2.0/#dfn-verifier) with enough information to determine if the provided data conforms to the provided schema.
> 
> All things considered, it would probably also be better to change that MUST to a SHOULD.


tl;dr if including a `credentialSchema` is a _can_ then the normative guidance is a _should_.

I personally do not see a problem with leaving the language as it was, but I defer to those more experienced than me with normative language.

Reply:
with other optional properties we followed the pattern of saying the property MAY be present, but if it is present, it MUST follow the guidance. See also [validity period](https://w3c.github.io/vc-data-model/#validity-period), [status](https://w3c.github.io/vc-data-model/#status), and `id`, `verifiableCredential`, and `proof` in [presentations](https://w3c.github.io/vc-data-model/#presentations-0)

Softening the MUST to a SHOULD makes the presence of the property not mean the same thing in all uses, which is substantially different than the previous wording, and it no longer matches the language in the rest of the spec. If there is no use case driving this change, I am opposed to it.

Reply:
It also means we don't need to test it... MUST requires us to test, SHOULD is optional to test.

Reply:
OK, reading this I am in favor of reverting my change. Reverted.